### ISSUE TITLE
Adding a score method for BayesianModel

### ISSUE THREAD

**Issue by aelnouby** [pgmpy/pgmpy#657]
@ankurankan  I am thinking of adding a score method for the BayesinModel class.

It may be like this `def score(self, X, y,):`

Where `y` is the correct targets (the column we usually drop)
and `X` is the data with  one variable missing (the one dropped to be predicted)

**Comment by khalibartan** [comment]
I cannot really say for this. But for me particularly its not that useful.

**Comment by aelnouby** [comment]
@khalibartan it is just it is very convenient for any one who has been using scikit-learn or similar libraries. after `fit` and `predict` someone usually would want to check `score` , i felt that way personally when i was trying the examples.

**Comment by khalibartan** [comment]
Yes indeed, but when I'm using bayesian models it is very rare that I have the correct output values. Also till now I have only used pgmpy for inference purpose on some data sets, but never for learning purpose, so for now it is not useful for me. Let see @ankurankan find it useful for not.
